### DISCUSSION TITLE
[PendingTimeout] pending timeout(>7200), evict by system

### DISCUSSION THREAD

**Discussion by fersebas** [PaddlePaddle/PaddleSpatial#289] [BaiduKDDCup2022] [answered]
Dear organizers,

We just have this errors but our models usually takes 40 miinutes at most in ai studio.

We have to submit again the models? Or it can be reevaluate after? I think the server is overloaded.

Thanks so much. (datateam-UCM)

**Comment by ramber1836** [comment, marked as answer]
You can resubmit first. If the error keeps occurring, we can reevaluate after.

**Reply by fersebas** [reply]
Ok, got it. thanks for the quick response

**Reply by gengdd** [reply]
> You can resubmit first. If the error keeps occurring, we can reevaluate after.

We also failed to submit. I don’t understand the resource allocation strategy of the server. All submissions later than us were successful, but we have pending Timeout. After we have computing resources, shouldn’t we be evaluated first?

**Reply by ramber1836** [reply]
OK. Please provide your team name and submission time.

**Reply by gengdd** [reply]
> 

We submitted the model zip at 17:12 today, but encountered Pending Timeout, so we resubmitted the model zip at 19:13. We estimated that the model should produce results within 30 minutes, but before the end of phase 3 It's still under evaluation,
Therefore, there is a high probability that the last submission will require a Pending Timeout. Team name: HIK

**Reply by fersebas** [reply]
Could you please reevaluate the submission  at time 2022-07-19 19:20. Our team is datateam-UCM. Thanks so much

**Reply by ramber1836** [reply]
> Could you please reevaluate the submission at time 2022-07-19 19:20. Our team is datateam-UCM. Thanks so much

Done

**Reply by fersebas** [reply]
Thanks ;)

**Reply by ramber1836** [reply]
> > 
> 
> We submitted the model zip at 17:12 today, but encountered Pending Timeout, so we resubmitted the model zip at 19:13. We estimated that the model should produce results within 30 minutes, but before the end of phase 3 It's still under evaluation, Therefore, there is a high probability that the last submission will require a Pending Timeout. Team name: HIK

It is under re-evaluating.

**Reply by wizzniu** [reply]
Our last submission at 17:02 is still evaluating, which seems to be very strange cause it‘s been over 10 hours .Could u plz check the backend to see what has happened? Team name is "wizzTeam".Thanks so much @ramber1836

**Reply by gengdd** [reply]
> > > 
> > 
> > 
> > We submitted the model zip at 17:12 today, but encountered Pending Timeout, so we resubmitted the model zip at 19:13. We estimated that the model should produce results within 30 minutes, but before the end of phase 3 It's still under evaluation, Therefore, there is a high probability that the last submission will require a Pending Timeout. Team name: HIK
> 
> It is under re-evaluating.
Thanks
